babel_monitor/src/lib.rs

```diff
@@ -59,6 +59,7 @@ pub struct Route {
     pub prefix: IpNetwork,
     pub metric: u16,
     pub refmetric: u16,
+    pub full_path_rtt: f32,
```

Comment:
FYI i chose `f32` for effortless parsing. Babeld operates on `int`s though and should we choose to follow its steps closer, we might want to parse the socket output "on foot".

Reply:
f32 is float right? I don't think that's going to parse correctly versus babel's 'thousands' format which isn't even close to real floating point.

Reply:
@jkilpatr It's actually pretty close, except no exp notation is at play. Babel simply "divides" the value by 1000 and pretends it's float by splitting the formatting:
```c
const char *
format_thousands(unsigned int value)
{
    static char buf[4][15];
    static int i = 0;
    i = (i + 1) % 4;
    snprintf(buf[i], 15, "%d.%.3d", value / 1000, value % 1000);
    return buf[i];
}
```

Reply:
do you think 'pretty close' is a one of the sources of the current inaccuracy?

Reply:
@jkilpatr Lol I just checked `ping` in Babel's tests vs. the RTT. I might've been barking up the wrong tree the whole time, as the value wildly diverge there too

Reply:
by 'this problem' do you mean parsing the thousands value and doing operations on it or just time measurement with the endpoints?

Reply:
I meant the measurement